Babylon/commands/api/datasets/apply.py

```diff
@@ -0,0 +1,70 @@
+import json
+import sys
+from logging import getLogger
+from pathlib import Path
+
+import click
+import yaml
+from click import command, option
+from mako.template import Template
+
+from Babylon.commands.api.datasets.service.api import DatasetService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import retrieve_state, wrapcontext
+from Babylon.utils.environment import Environment
+from Babylon.utils.response import CommandResponse
+from Babylon.utils.yaml_utils import yaml_to_json
+
+logger = getLogger("Babylon")
+env = Environment()
+
+
+@command()
+@wrapcontext()
+@pass_azure_token("csm_api")
+@option("--organization-id", "organization_id", type=str)
+@option("--dataset-id", "dataset_id", type=str)
+@retrieve_state
+def apply(state: dict, azure_token: str, organization_id: str, dataset_id: str):
+    service_state = state["services"]
+    stream = click.get_text_stream("stdin")
+    data = stream.read()
```

Comment:
the way to check out if 'stdin' data exists has changed, you can see it in the #285